bricks/basic-bricks/dist-types/app-bar-setting/index.d.ts

```
@@ -0,0 +1,12 @@
+import React from "react";
+import type React from "react";
```

Biome (2.4.4)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)